content/blog/2024/10/kubewarden-1-17-release.md

```diff
@@ -0,0 +1,166 @@
+to using existing policies, or for example using CEL via cel-policy).
+
+However, while ordinary policies can include mutation logic to modify resources
+during admission, policy groups are limited to validation only.
+
+Configuration for policy groups is similar to that of ordinary policies. The
+difference is the addition of the `expression`, `message`, and `policies` fields, as
+well as the declaration of context-aware rules in a different location.
+
+Here is an example of a ClusterAdmissionPolicyGroup that makes sure a Service
+selector is unique or owned by a specific team:
+
+```yaml
+---
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicyGroup # or AdmissionPolicyGroup
+metadata:
+  name: unique-service-selector
+spec:
+  rules:
+    - apiGroups: [""]
+      apiVersions: ["v1"]
+      resources: ["services"]
+      operations: ["CREATE", "UPDATE"]
+  policies:
+    unique_service_selector:
+      module: registry://ghcr.io/kubewarden/policies/unique-service-selector-policy:v0.1.0
+      contextAwareResources:
+        - apiVersion: v1
+          kind: Service
+      settings:
+        app.kubernetes.io/name: MyApp
+    owned_by_foo_team:
+      module: registry://ghcr.io/kubewarden/policies/safe-annotations:v0.2.9
+      settings:
+        mandatory_annotations:
+          - owner
+        constrained_annotations:
+          owner: "foo-team"
+  expression: "unique_service_selector() || (!unique_service_selector() && owned_by_foo_team())"
+  message: "the service selector is not unique or the service is not owned by the foo team"
+```
+
+As usual, the CRDs are documented in our
+[reference](https://docs.kubewarden.io/reference/CRDs#resource-types) docs. And
+you can read their
+[explanation](https://docs.kubewarden.io/explanations/policy-groups) and an
+in-depth [how-to](https://docs.kubewarden.io/next/howtos/policy-groups) on the
+docs for these resources.
+
+Stay tuned for a new deep dive blog post!
+
+## shortNames for our CRDs
+
+Kubernetes supports short names for CRDs, and with the new PolicyGroups
+feature, we think it's the perfect moment to include them. Short names simplify
+day-to-day operations. For example, to list ClusterAdmissionPolicies:
+
+```
+kubectl get cap
+NAME                        POLICY SERVER   MUTATING   BACKGROUNDAUDIT   MODE      OBSERVED MODE   STATUS   AGE
+do-not-run-as-root          default         true       true              monitor   monitor         active   3d
+do-not-share-host-paths     default         false      true              monitor   monitor         active   3d
+...
+```
+
+That's a lot better, right?
+
+Here's our shortened resource types:
+
+| Resource                     | shortName |
+| ---------------------------- | --------- |
+| AdmissionPolicies            | **ap**    |
+| ClusterAdmissionPolicies     | **cap**   |
+| AdmissionPolicyGroups        | **apg**   |
+| ClusterAdmissionPolicyGroups | **capg**  |
+| PolicyServers                | **ps**    |
```

Comment:
As we discussed during the planning, this will land in the next release. Thus, we need to remove this section.

Reply:
Dropped with https://github.com/kubewarden/kubewarden.io/pull/265/commits/7a41a98c9f6e8c3a2dfe224dd1b2b6d1942ed951.